Makefile

```diff
@@ -73,3 +87,38 @@ sonar:
 .PHONY: sonar-pr-analysis
 sonar-pr-analysis:
 	mvn sonar:sonar -P sonar-pr-analysis
+
+##### Start of dependency-check block to be put at bottom of Makefile
+.PHONY: dependency-check
+dependency-check:
+	@ if [ -d "$(DEPENDENCY_CHECK_SUPPRESSIONS_HOME)" ]; then   \
+		suppressions_home="$${DEPENDENCY_CHECK_SUPPRESSIONS_HOME}"; \
+	fi; \
+	if [ ! -d "$${suppressions_home}" ]; then   \
+		suppressions_home_target_dir="./target/dependency-check-suppressions";  \
+		if [ -d "$${suppressions_home_target_dir}" ]; then  \
+			suppressions_home="$${suppressions_home_target_dir}";   \
+			else    \
+		mkdir -p "./target";    \
+			git clone $(dependency_check_suppressions_repo_url) "$${suppressions_home_target_dir}" && \
+			suppressions_home="$${suppressions_home_target_dir}"; \
+			if [ -d "$${suppressions_home_target_dir}" ] && [ -n "$(dependency_check_suppressions_repo_branch)" ]; then \
+			cd "$${suppressions_home}"; \
+			git checkout $(dependency_check_suppressions_repo_branch); \
+			cd -; \
+			fi; \
+		fi; \
+	fi; \
+	suppressions_path="$${suppressions_home}/suppressions/$(dependency_check_base_suppressions)"; \
+	if [  -f "$${suppressions_path}" ]; then \
+		cp -av "$${suppressions_path}" $(suppressions_file); \
+		mvn org.owasp:dependency-check-maven:check -DfailBuildOnCVSS=$(dependency_check_minimum_cvss) -DassemblyAnalyzerEnabled=$(dependency_check_assembly_analyzer_enabled) -DsuppressionFiles=$(suppressions_file); \
+		else \
+		printf -- "\n ERROR Cannot find suppressions file at '%s'\n" "$${suppressions_path}" >&2; \
+		exit 1; \
+	fi
+
+.PHONY: security-check
```

Comment:
@foman-ch  it's pointing at the dependency-check job and probably good to keep in for backwards compatibility?

Reply:
This seems as though it may be the issue but if you look at this [PR](https://github.com/companieshouse/dissolution-api/pull/684/files) it has that in there but no security warnings.

Reply:
@dhawes-ch yep, I mean to keep this and delete the security-check block on line 33 :)

Reply:
Ahh misunderstood, good spot

Reply:
Done.